.drone.yml

```diff
@@ -488,6 +498,7 @@ depends_on:
   - e2e-kubernetes-1.18
   - e2e-kubernetes-1.19
   - e2e-kubernetes-1.20
+  - e2e-kubernetes-v1.21
```

Comment:
Be consistent please

Reply:
@angelbarrera92 the pipeline stage name is `e2e-kubernetes-v1.21`. thought it was a naming change convention you were adding. :)

Reply:
I will fix it in both places then